core/block_svg.js

```diff
@@ -1220,7 +1235,7 @@ BlockSvg.prototype.setDeleteStyle = function(enable) {
 };
 
 
-// Overrides of functions on Blockly.Block that take into account whether the
+// Overrides of functions on Block that take into account whether the
```

Comment:
I'm not sure if this one should be changed or not. Your call since you'll be better at knowing what's confusing.

Reply:
Done